src/dune_lang/action.mli

```diff
@@ -120,7 +120,10 @@ type t =
   | Diff of (String_with_vars.t, String_with_vars.t) Diff.t
   | No_infer of t
   | Pipe of Outputs.t * t list
-  | Cram of String_with_vars.t
+  | Cram of
```

Comment:
I would make this a tuple rather than a record to keep it more inline with the rest of the entries.

Reply:
In an earlier iteration the `shell_spec` field was `String_with_vars.t` so I made it a record to avoid confusion. Now it makes more sense to make it a tuple for consistency.

addressed in 716b5fcc1